src/nbl/video/CVulkanConnection.h

```diff
@@ -0,0 +1,59 @@
+#ifndef __NBL_C_VULKAN_CONNECTION_H_INCLUDED__
+#define __NBL_C_VULKAN_CONNECTION_H_INCLUDED__
+
+#include <volk.h>
+#include "nbl/video/IAPIConnection.h"
+#include "nbl/video/surface/CSurfaceVKWin32.h"
+
+namespace nbl {
+namespace video
+{
+
+class CVulkanConnection final : public IAPIConnection
+{
+public:
+    CVulkanConnection()
+    {
+        VkResult result = volkInitialize();
+        assert(result == VK_SUCCESS);
+
+        VkApplicationInfo app;
+        app.sType = VK_STRUCTURE_TYPE_APPLICATION_INFO;
+        app.apiVersion = VK_MAKE_VERSION(1, 1, 0);
```

Comment:
can you do a little research and find out if there are VK 1.1 implementations in the wild? Which have not updgraded to 1.2 in the meantime?

Reply:
Assign to @sadiuk

Reply:
@Devsh-Graphics-Programming/seniors there are, looks like android is not yet on Vulkan 1.2: https://developer.android.com/about/dashboards#Vulkan

Reply:
data collected on Jan 8, 2021

Reply:
ok stays as is